src/main/java/seedu/duke/template/LcTemplate.java

```diff
@@ -11,8 +11,8 @@
                                                 + "\t|             |\n"
                                                 + "\t|             |\n"
                                                 + "\t+----+V_ac+---+\n";
-    private Inductor inductor;
-    private Capacitor capacitor;
+    private final Inductor inductor;
+    private final Capacitor capacitor;
```

Comment:
Any particular reason as to why you declared these as final? Because when a variable is declared with final keyword, its value can’t be modified, essentially, a constant.

Reply:
Okay good point - I didn't consider that. I modified the declaration as per IntelliJ's suggestion because if I remember correctly, it flagged a warning (but not an error).

But like Wira pointed out, they will still work when we declare them as `final` right?